.TAOS-CI/taos/auto-unittest.sh

```diff
@@ -42,11 +42,11 @@ do
     # Parsing result 
     test_flag=0
     while IFS='' read -r line || [[ -n "$line" ]]; do
-        if [[ $line =~  "./unittest_common" ]]; then
+        if [[ $line =~  "./tests/unittest_common" ]]; then
             test_flag=1
         fi
 
-        if [[ $line =~ "./testAll.sh" ]]; then
+        if [[ $line =~ "+ ssat" ]]; then
```

Comment:
Is it confirmed that this regex works for our log? (have you tested if test_flag=2 is set after using this?) If so, assume LGTM.

Reply:
yes I tested. '$' doesn't work. so removed it.

Reply:
yes I tested.